app/src/main/kotlin/za/foundation/praekelt/mama/app/App.kt

```diff
@@ -2,24 +2,60 @@ package za.foundation.praekelt.mama.app
 
 import android.app.Application
 import com.raizlabs.android.dbflow.config.FlowManager
+import com.squareup.otto.Bus
+import com.squareup.otto.Subscribe
+import org.jetbrains.anko.AnkoLogger
+import rx.Observable
 import za.foundation.praekelt.mama.inject.component.ApplicationComponent
 import za.foundation.praekelt.mama.inject.component.DaggerApplicationComponent
 import za.foundation.praekelt.mama.inject.module.ApplicationModule
+import java.util.HashMap
 
 /**
  * Created by eduardokolomajr on 2015/06/25.
  * Application class
  */
 
-class App:Application(){
+class App : Application(), AnkoLogger {
+    val bus: Bus
+    val observables: MutableMap<String, List<Observable<out Any>>>
+
+    init {
+        bus = Bus()
+        observables = HashMap<String, List<Observable<out Any>>>()
+    }
+
     override fun onCreate() {
-        super.onCreate()
+        super<Application>.onCreate()
         FlowManager.init(this)
+        bus.register(this)
+    }
+
+    override fun onLowMemory() {
```

Comment:
This method is not called very reliably in my experience and the way it is used differs across Android versions. Best be sure this is going to work/is necessary.